onprc_ehr/resources/web/onprc_ehr/model/sources/NecropsyRequest.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2013-2014 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+//Created: 1-21-2021  R.Blasa
+EHR.model.DataModelManager.registerMetadata('PathTissues', {
+    allQueries: {
+    },
+    byQuery: {
+        'study.encounters': {
+            type: {
+                defaultValue: 'Necropsy',
+                hidden: true
+            },
+            chargetype: {
+                defaultValue: 'DCM: Pathology Services',
+                hidden: true
+            },
+            date: {
+                xtype: 'xdatetime',
+                header: 'Start Date',
+                extFormat: 'Y-m-d H:i',
+                defaultValue: (new Date()).format('Y-m-d 8:0')
```

Comment:
This is the same unexpected date format that I asked Lakshmi about. Is the goal to default to 8:00AM?